lib/main.coffee

```diff
@@ -77,9 +88,10 @@ module.exports =
             '--output-format=text'
           ]
           if @rcFile
-            args.push "--rcfile=#{@rcFile}"
+            rcFile = @rcFile.replace(/%p/g, projDir).replace(/%f/g, path.dirname(file))
```

Comment:
IMO, global replacement makes sense for `pythonPath` because we may want to add multiple directories in the project to `PYTHONPATH` e.g., `%p/foo:%p/bar`.  I cannot think of a use case for specifying project or file dir multiple times in the `rcFile` path.